xbmc/interfaces/python/PythonInvoker.h

```diff
@@ -58,9 +58,20 @@ class CPythonInvoker : public ILanguageInvoker
   std::string m_sourceFile;
   CCriticalSection m_critical;
 
+protected:
+  template<size_t NUMBER>
```

Comment:
`NUMBER` isn't a great name, `SIZE` or `COUNT` would be more suitable.

Reply:
This part is removed.